training/backends/kubernetes.js

```diff
@@ -161,10 +182,35 @@ module.exports = async function execTask(job, spec) {
                                     memory: (Config.TRAINING_MEMORY_USAGE + 100) + 'Mi'
```

Comment:
By the way, one problem I noticed is that Config.TRAINING_MEMORY_USAGE is not actually passed to the node process, so we would still hit Javascript OOM. It needs to be passed as `--max_old_space_size`, and needs to be passed before the `main.js` script, so we need to be override the image entrypoint.